sim/druid/lacerate.go

```diff
@@ -0,0 +1,131 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (druid *Druid) registerLacerateSpell() {
+	if !druid.HasRune(proto.DruidRune_RuneLegsLacerate) {
+		return
+	}
+	initialDamageMul := 0.0
+	hasGore := druid.HasRune(proto.DruidRune_RuneHelmGore)
+
+	switch druid.Ranged().ID {
+	case IdolOfCruelty:
+		initialDamageMul += .07
+	}
+	rageMetrics := druid.NewRageMetrics(core.ActionID{SpellID: 431446})
+
+	druid.Lacerate = druid.RegisterSpell(Bear, core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 414644},
+		SpellSchool: core.SpellSchoolPhysical,
+		SpellCode:   SpellCode_DruidLacerate,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags:       SpellFlagOmen | core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
+
+		RageCost: core.RageCostOptions{
+			Cost:   10,
+			Refund: 0.8,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: core.GCDDefault,
+			},
+			IgnoreHaste: true,
+		},
+
+		DamageMultiplier: 1,
+		ThreatMultiplier: 3.33,
+		// TODO: Berserk 3 target lacerate cleave - Saeyon
```

Comment:
My suggestion for this would be to bake a target count into lacerate's code and default it to. Maybe you can use an attribute on the druid to track the current target count that lacerate should hit. Sounds like there's two different effects that make lacerate affect multiple targets?